index.js

```diff
@@ -12,6 +13,10 @@ module.exports = function (numbers, opts) {
 	var finiteNumbers = numbers.filter(isFinite);
 	var min = opts.min || Math.min.apply(null, finiteNumbers);
 	var max = opts.max || Math.max.apply(null, finiteNumbers);
+	var style = opts.style || '';
+	if (style === 'fire') {
+		ticks = [colors.fg.getRgb(5, 5, 3) + '▁', colors.fg.getRgb(5, 5, 4) + '▂', colors.fg.getRgb(5, 5, 0) + '▃', colors.fg.getRgb(5, 4, 0) + '▄', colors.fg.getRgb(5, 3, 0) + '▅', colors.fg.getRgb(5, 2, 0) + '▆', colors.fg.getRgb(5, 1, 0) + '▇', colors.fg.getRgb(5, 0, 0) + '█'];
```

Comment:
I wouldn't copy the sparks all over again, just create a color array and add the colors in the map function below if `style` is set to `fire`.

```js
var fire = [colors.fg.getRgb(5, 5, 3), colors.fg.getRgb(5, 5, 4), colors.fg.getRgb(5, 5, 0), ...];

// ...

return (style === 'fire' ? fire[tickIndex] : '') + ticks[tickIndex];
```